src/issueFinder.ts

```diff
@@ -56,6 +58,10 @@ export class IssueFinder {
         return issuesByRepo;
     }
 
+    private checkIfAssigned(gitHubIssue: GitHubIssue): boolean {
```

Comment:
To increase readability, given the return of a boolean, you my consider changing this method name to `isUnassigned` since it'll return true if unassigned, false otherwise.